### PR TITLE
fix(Login): remove white spaces before send to BE

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- add before submit in login a trim function to remove white spaces in user name

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ const Login = ({ location, dependencies: { dopplerLegacyClient, sessionManager,
   const onSubmit = async (values, { setSubmitting, setErrors }) => {
     try {
       const result = await dopplerLegacyClient.login({
-        username: values[fieldNames.user],
+        username: values[fieldNames.user].trim(),
```
Comment (cbernat) [inline src/components/Login/Login.js:116]: @andresmoschini I summon you, any better idea than doing this? back end is taking litteraly what we send, if we send spaces it takes them as part of the email. In old login the trimming was done in the view.

**Comment by cbernat** [comment]
If we don't have any opinions yet, I think it's better to merge. 🚀

**Comment by fcoronelmakingsense** [comment]
testing here http://cdn.fromdoppler.com/doppler-webapp/build2304/#/login

File: src/components/Login/Login.js (hunk 1 above)
Comment (andresmoschini) [reply src/components/Login/Login.js:116]: Sorry, I did not saw this one. 

I think that it is fine in this way, it cannot be _null_, right?

**Pull request merged**